modules/installation-creating-gcp-worker.adoc

```diff
@@ -127,6 +127,14 @@ file.
 $ gcloud deployment-manager deployments create ${INFRA_ID}-worker --config 06_worker.yaml
 ----
 
+[.small]
+--
+1. To use a GCP Marketplace image, specify the offer to use:
```

Comment:
@mjpytlak Are these sites supposed to go to a code sample? Just checking since I am not familiar with these offers. Also, is 'offer' a standard reference for these options? Thanks!

Reply:
@tmalove Normally I would be these elsewhere, likely a procedure about specifying how specify an image for a worker node. However, this particular update is for UPI. UPI doc is deliberately not prescriptive, as the installation method is geared towards folks who want to maintain the underlying infrastructure of their cluster. In the context of the overall deployment, this is the best spot for them.